stacks/upload-api-stack.js

```diff
@@ -141,6 +141,7 @@ export function UploadApiStack({ stack, app }) {
           STRIPE_PUBLISHABLE_KEY: process.env.STRIPE_PUBLISHABLE_KEY ?? '',
           DEAL_TRACKER_DID: process.env.DEAL_TRACKER_DID ?? '',
           DEAL_TRACKER_URL: process.env.DEAL_TRACKER_URL ?? '',
+          INDEXER_REGION: EIPFS_MULTIHASHES_SQS_ARN.includes('us-west-2') ? 'us-west-2' : 'us-east-2'
```

Comment:
nit: We always have one right? Why not parse it out from the ARN? https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html#arns-syntax

Reply:
it is actually always `west-2`, but I can parse it yes